prog/vm/nexus.rb

```diff
@@ -225,6 +239,26 @@ def before_run
 
     host.sshable.cmd(command)
 
+    hop_wait_for_slice
+  end
+
+  label def wait_for_slice
+    unless vm.vm_host_slice.nil?
+      if !vm.vm_host_slice.enabled
```

Comment:
Can we switch this to `if vm.vm_host_slice.enabled` and reverse the order of the branches?